config.yaml

```diff
@@ -10,6 +10,9 @@ menu:
     - name: News
       URL: news
       weight: 1
+    - name: Resources
+      URL: resources
+      weight: 1
```

Comment:
Should news and resources have the same weight?

Reply:
Hi @RichardLitt :
The weight parameter is used to control the ordering of items, such as menu entries, sections, or other elements. It's okay to have the same weight for news and resources which means they will be displayed first, likely in the order they are listed in the YAML file.

However, to avoid confusion here it's better to number them respectively as in the the header.